tests/test_core.py

```diff
@@ -316,6 +316,7 @@ def test_invalid_names_raise(self):
 
     def test_empty_labels_list(self):
         h = Histogram('h', 'help', [], registry=self.registry)
+        h.observe(0)
```

Comment:
This changes the test

Reply:
It does, but the test still fails if the change from commit e5e73e52 is reversed. It's really testing the constructor not recorded values, which is covered by `TestHistogram.test_histogram`.

Is there a better way of asserting "this isn't a `_LabelWrapper` that needs `labels()` called before using? I don't really want to use `isinstance()`.

Reply:
You could not assign to a variable

Reply:
Okay, I wanted a better assertion but if you're fine with just checking the registry.